Sources/Private/Utility/Extensions/MathKit.swift

```diff
@@ -25,6 +25,20 @@ extension Double {
 
 extension CGFloat: Interpolatable {
 
+  // MARK: Public
+
+  public func interpolateTo(
```

Comment:
If we're making these interpolation methods part of the public API should they get some documentation? I'm not really sure how these are intended to be used as-is